hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -279,15 +285,22 @@ public Instant parentConsensusTimestamp() {
         return parentConsensus;
     }
 
+    @Override
+    public long transactionFee() {
+        return transactionFee;
+    }
+
     /**
      * Sets the consensus transaction fee.
      *
      * @param transactionFee the transaction fee
      * @return the builder
      */
     @NonNull
+    @Override
```

Comment:
Nit: The order of these two should be `@Override @NonNull` to match the rest of the overridden methods in this class

Reply:
True :-(